Library/Formula/libstfl.rb

```diff
@@ -13,21 +13,24 @@ class Libstfl < Formula
   end
 
   depends_on :python => :optional
+  depends_on "ruby"
```

Comment:
You might want to add an explanation of why we're enforcing our Ruby rather than using the special `:ruby` dep. It should also be safe to remove the `args << "FOUND_RUBY = 0"` line if we're enforcing using ours with no opt-out.